package.json

```diff
@@ -17,7 +17,7 @@
     "lint:js": "eslint .",
     "lint:license": "node scripts/license-checker",
     "format": "eslint . --fix",
-    "test": "lerna run test --stream --scope=$SCOPE",
+    "test": "lerna run test --stream --scope=$SCOPE --no-prefix",
```

Comment:
Since we run the tests based on lerna scope and we already are aware of the package name beforehand. Instead its useful to print out the test labels like `supported` and `failsafe` which made more sense. check this 
https://github.com/elastic/apm-agent-js-base/blob/01f17539d5736c298bcfece992aa7780cc28e580/packages/rum/run-script.js#L158